src/Server/HTTP/sendExceptionToHTTPClient.cpp

```diff
@@ -26,15 +26,24 @@ void drainRequestIfNeeded(HTTPServerRequest & request, HTTPServerResponse & resp
         && (request.getChunkedTransferEncoding() || request.hasContentLength())
         && response.getKeepAlive())
     {
-        try
+        /// If the client expects 100 Continue, but we never sent it, don't attempt to read the body and
```

Comment:
In general I do not understand this part of code. What prevents us from reusing session at error?